lte/gateway/docker/ghz/docker-compose.yaml

```diff
@@ -0,0 +1,147 @@
+---
```

Comment:
If this file is largely similar to the [base docker-compose](https://github.com/magma/magma/blob/master/lte/gateway/docker/docker-compose.yaml) file to startup all the services, would it help if we use the same docker-compose file as above, but only startup selective set of containers (perhaps using the -d option with the container name?)

This would ensure that we have a single source file to maintain and capture changes for the common dev-ops, scale-test and other use-cases.

Reply:
This is indeed really similar to our docker AGW `docker-compose` file, the only important change is the definition of the `image:` on both services (`image: ${DOCKER_REGISTRY}/ghz_gateway_python:${IMAGE_VERSION}`) 

is there any way we could use the base file and just change this value? maybe defining a variable on the `.env` file?

Reply:
How about using the [ansible module for docker-compose](https://docs.ansible.com/ansible/latest/collections/community/docker/docker_compose_module.html) which supports the 'services' option which allows the user to specify only a subset of services to start? The link includes an example where you use the docker-compose module within and specify a set of services and any custom changes.

Reply:
Tried that but this requires `docker-compose` python dependency to be installed as well, and looks like that ansible module does not support changing the `image` (which is different from the base `docker-compose.yaml` file)